test/e2e/dynamic-import-app/test.js

```diff
@@ -175,13 +175,6 @@ test('`fusion build` app with dynamic imports integration', async () => {
     'one extra script after loading new route'
   );
 
-  t.ok(
```

Comment:
The reason this crossorigin logic exists is because of:
https://github.com/webpack/webpack/issues/6972#issue-311819636

So the test is verifying that when not using a CDN, no crossorigin attributes are set. I don't think we want to delete these tests. I think this test should remain unchanged.

Instead, there *are* tests that does use a CDN, and we should verify that crossorigin attributes *are* set. As you point out, these assertions would probably currently fail.

Reply:
added tests for CDN case if we want to go that route.